src/Groups/GAPGroups.jl

```diff
@@ -1304,6 +1304,39 @@ an exception is thrown if `G` is not solvable.
 end
 
 
+################################################################################
+#
+# Subgroups defined by properties
+#
+################################################################################
+
+"""
+    subgroup_property(prop::Function, G:: PermGroup)
```

Comment:
I find this name confusing (also in GAP). Maybe `subgroup_by_property`? Though even then it somewhat clashes with the name `sub` (not *subgroup*). But `sub_by_property` doesn't really sound better. Ah well.

Reply:
O.k.